Tests/FluentAssertions.Specs/Types/TypeAssertionSpecs.HaveConstructor.cs

```diff
@@ -21,9 +21,9 @@ public void When_asserting_a_type_has_a_constructor_which_it_does_it_succeeds()
             // Act
             Action act = () =>
                 type.Should()
-                    .HaveConstructor(new Type[] { typeof(string) })
+                    .HaveConstructor(new[] { typeof(string) })
                     .Which.Should()
-                        .HaveAccessModifier(CSharpAccessModifier.Private);
+                    .HaveAccessModifier(CSharpAccessModifier.Private);
```

Comment:
I liked the previous indentation as it makes it more clear that `HaveAccessModifier` binds to the constructor and not the type.

An alternative would be
```cs
.Which.Should().HaveAccessModifier(CSharpAccessModifier.Private);
```

Reply:
That may be true, but technically the auto-indention is correct.